pkg/loadbalancer/stream/loadbalancer.go

```diff
@@ -64,6 +65,13 @@ func New(stream *nspAPI.Stream, targetRegistryClient nspAPI.TargetRegistryClient
 		nfqueue:                    nfqueue,
 		pendingTargets:             make(map[int]types.Target),
 	}
+	// first enable kernel's IP defrag except for the interfaces facing targets
+	// (defrag is needed by Flows to match rules with L4 information)
+	loadBalancer.defrag, err = NewDefrag(types.InterfaceNamePrefix)
```

Comment:
Having NewDefrag here means the table, chains and rules for defrag will be created each time a stream is created (so multiple times if there is multiple streams). Was this the expected behavior?

Reply:
Yes, I checked and table/chain create will not return an error if the table/chain is already present, and creates what's been asked only if it does not exist.

Reply:
ok, otherwise this could be moved directly in cmd/load-balancer

Reply:
keeping the code at its original place for now until multiple conduits will get introduced